packages/nextjs/src/config/webpackPluginOptions.ts

```diff
@@ -11,37 +11,43 @@ export function getWebpackPluginOptions(
   buildContext: BuildContext,
   sentryBuildOptions: SentryBuildOptions,
 ): SentryWebpackPluginOptions {
-  const { buildId, isServer, config: userNextConfig, dir: projectDir, nextRuntime } = buildContext;
+  const { buildId, isServer, config: userNextConfig, dir, nextRuntime } = buildContext;
 
   const prefixInsert = !isServer ? 'Client' : nextRuntime === 'edge' ? 'Edge' : 'Node.js';
 
-  const distDirAbsPath = path.join(projectDir, (userNextConfig as NextConfigObject).distDir || '.next'); // `.next` is the default directory
+  // We need to convert paths to posix because Glob patterns use `\` to escape
+  // glob characters. This clashes with Windows path separators.
+  // See: https://www.npmjs.com/package/glob
+  const projectDir = dir.replace(/\\/g, '/');
+  // `.next` is the default directory
+  const distDir = (userNextConfig as NextConfigObject).distDir?.replace(/\\/g, '/') ?? '.next';
```

Comment:
Wondering if we could use `path.posix.normalize` instead of `replace` and whether it would make this a bit more robust 🤔

Reply:
Normalize uses the platform specific separator, so even with `.posix` it would keep backslashes.

e.g.
```
path.posix.normalize('C:\\a\\b\\c')
-> 'C:\\a\\b\\c'
```